src/main/java/com/github/nicholasmoser/gnt4/seq/ext/SeqAssembler.java

```diff
@@ -0,0 +1,1171 @@
+            new SimpleEntry<>("gpr19", (byte)0x13),
+            new SimpleEntry<>("cr_companion", (byte)0x13),
+            new SimpleEntry<>("gpr20", (byte)0x14),
+            new SimpleEntry<>("ctr", (byte)0x14),
+            new SimpleEntry<>("gpr21", (byte)0x15),
+            new SimpleEntry<>("cr", (byte)0x15),
+            new SimpleEntry<>("gpr22", (byte)0x16),
+            new SimpleEntry<>("stored_pc", (byte)0x16),
+            new SimpleEntry<>("gpr23", (byte)0x17),
+            new SimpleEntry<>("sp", (byte)0x17),
+            new SimpleEntry<>("seq_p_sp0", (byte)0x18),
+            new SimpleEntry<>("seq_p_sp1", (byte)0x19),
+            new SimpleEntry<>("seq_p_sp2", (byte)0x1A),
+            new SimpleEntry<>("seq_p_sp3", (byte)0x1B),
+            new SimpleEntry<>("seq_p_sp4", (byte)0x1C),
+            new SimpleEntry<>("seq_p_sp5", (byte)0x1D),
+            new SimpleEntry<>("final_instruction_seq", (byte)0x1D),
+            new SimpleEntry<>("seq_p_sp6", (byte)0x1E),
+            new SimpleEntry<>("seq_p_sp7", (byte)0x1F),
+            new SimpleEntry<>("seq_p_sp8", (byte)0x20),
+            new SimpleEntry<>("mot", (byte)0x20),
+            new SimpleEntry<>("animation", (byte)0x20),
+            new SimpleEntry<>("seq_p_sp9", (byte)0x21),
+            new SimpleEntry<>("movement", (byte)0x21),
+            new SimpleEntry<>("seq_p_sp10", (byte)0x22),
+            new SimpleEntry<>("chr_p_field_0x4C", (byte)0x22),
+            new SimpleEntry<>("seq_p_sp11", (byte)0x23),
+            new SimpleEntry<>("seq_p_sp12", (byte)0x24),
+            new SimpleEntry<>("seq_p_sp13", (byte)0x25),
+            new SimpleEntry<>("seq_p_sp14", (byte)0x26),
+            new SimpleEntry<>("chr_p", (byte)0x26),
+            new SimpleEntry<>("seq_p_sp15", (byte)0x27),
+            new SimpleEntry<>("foe_chr_p", (byte)0x27),
+            new SimpleEntry<>("seq_p_sp16", (byte)0x28),
+            new SimpleEntry<>("seq_p_sp17", (byte)0x29),
+            new SimpleEntry<>("seq_p_sp18", (byte)0x2A),
+            new SimpleEntry<>("seq_p_sp19", (byte)0x2B),
+            new SimpleEntry<>("seq_p_sp", (byte)0x2B),
+            new SimpleEntry<>("seq_p_sp20", (byte)0x2C),
+            new SimpleEntry<>("seq_p_sp21", (byte)0x2D),
+            new SimpleEntry<>("seq_p_sp22", (byte)0x2E),
+            new SimpleEntry<>("seq_p_sp23", (byte)0x2F),
+            new SimpleEntry<>("seq_file", (byte)0x2F),
+            new SimpleEntry<>("hitbox_identity_matrix", (byte)0x30),
+            new SimpleEntry<>("controllers", (byte)0x32),
+            new SimpleEntry<>("primary_controller", (byte)0x33),
+            new SimpleEntry<>("display", (byte)0x34),
+            new SimpleEntry<>("save_data", (byte)0x39),
+            new SimpleEntry<>("debug_mode", (byte)0x3A),
+            new SimpleEntry<>("pause_game", (byte)0x3B),
+            new SimpleEntry<>("game_info", (byte)0x3C),
+            new SimpleEntry<>("unused", (byte)0x3D));
```

Comment:
Pretty cool to actually see these all laid out like this, it didn't really occur to me that they were all in a row like that